geode-protobuf/src/main/java/org/apache/geode/internal/serialization/registry/SerializationCodecRegistry.java

```diff
@@ -27,15 +27,14 @@
 public class SerializationCodecRegistry {
   private HashMap<SerializationType, TypeCodec> codecRegistry = new HashMap<>();
 
-  public SerializationCodecRegistry() throws CodecAlreadyRegisteredForTypeException {
+  public SerializationCodecRegistry() {
     ServiceLoader<TypeCodec> typeCodecs = ServiceLoader.load(TypeCodec.class);
     for (TypeCodec typeCodec : typeCodecs) {
       register(typeCodec.getSerializationType(), typeCodec);
```

Comment:
I think we should handle the checked exception here, rather than passing it up for others to deal with.
Maybe we do
`try {
        register(typeCodec.getSerializationType(), typeCodec);
      } catch (CodecAlreadyRegisteredForTypeException e) {
        throw new GemFireConfigException("Duplicate Codec Registration.", e);
      }`